v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,127 @@
+---
+
+CockroachDB supports parallel execution of independent [`INSERT`](insert.html), [`UPDATE`](update.html), [`UPSERT`](upsert.html), and [`DELETE`](delete.html) statements within a single transaction. Parallel execution of independent SQL statements reduces the overall execution time of multiple SQL statements, thereby improving query performance.
+
+<div id="toc"></div>
+
+## Why Use Parallel Statement Execution
+
+Applications usually issue multiple SQL statements within a single transaction. SQL engines normally execute each SQL statement sequentially. Because the queries are executed locally, the results can be delivered fast. But CockroachDB being a distributed database, execution of queries takes longer than normal SQL engines because of high round-trip latencies required for consensus. To counteract the issue and optimize query performance, CockroachDB supports parallel execution of independent SQL statements. Thus, despite the higher latency of single data-modifying statements, the overall latency of multiple such statements can be reduced.
+
+## How Parallel Statement Execution Works
+
+SQL statements within a single transaction can be executed in parallel if the statements are independent. Normal execution of SQL statements returns a successful return value immediately from the server. After the return value is received, the next SQL statement is executed. Appending the `RETURNING NOTHING` clause with independent SQL statements prevents the server from sending return values and allows CockroachDB to execute the statements in parallel.
+
+To execute statements in parallel, append the `RETURNING NOTHING` suffix for the independent SQL statements.
+
+### Error message mismatch
+
+It is possible to see a mismatch in the error message being displayed and the transaction being executed. Consider a scenario where a dependent statement is executed after a batch of independent statements, and one of the independent statements results in an error. Now because the `RETURNING NOTHING` clause is used with the independent statements, CockroachDB cannot return the error for the independent statement. The error message for the independent statement is then reported on the next dependent statement. Thus you might see a mismatch with the error message to the statement being executed. 
+
+On encountering the error, CockroachDB will stop the execution of all parallel statements and [abort and retry the transaction](transactions.html#error-handling). So even an error message mismatch occurs, the result will be the same as aborting the transaction.
+
+### `RETURNING NOTHING` clause appended to dependent statements
+
+If two consecutive statements are not independent, and yet a `RETURNING NOTHING` clause is added to the statements, CockroachDB will detect the dependence and serialize the execution. You can thus use the `RETURNING NOTHING` clause with SQL statements without worrying about their dependence.
+
+### Delayed execution of dependent statements
```

Comment:
I think what you mean here is synchronous (returning results) vs. asynchronous (running in the background without returning results), not dependent vs independent.